src/ui/DynamicHierarchicalFacetSearch/DynamicHierarchicalFacetSearch.ts

```diff
@@ -0,0 +1,62 @@
+import { IDynamicHierarchicalFacet } from '../DynamicHierarchicalFacet/IDynamicHierarchicalFacet';
+import { HierarchicalFacetSearchController } from '../../controllers/HierarchicalFacetSearchController';
+import { Combobox } from '../Combobox/Combobox';
+import { l } from '../../strings/Strings';
+import { IFacetSearchResponse } from '../../rest/Facet/FacetSearchResponse';
+import { IComboboxValue } from '../Combobox/ComboboxValues';
+import { DynamicHierarchicalFacetSearchValue } from './DynamicHierarchicalFacetSearchValue';
+import { DynamicHierarchicalFacetSearchValueRenderer } from './DynamicHierarchicalFacetSearchValueRenderer';
+import 'styling/DynamicHierarchicalFacetSearch/_DynamicHierarchicalFacetSearch';
+
+export class DynamicHierarchicalFacetSearch {
```

Comment:
This class is heavily inspired by [`DynamicFacetSearch`](https://github.com/coveo/search-ui/blob/master/src/ui/DynamicFacetSearch/DynamicFacetSearch.ts).